inc/deploygroup.class.php

```diff
@@ -114,7 +114,7 @@ public function defineTabs($options = [])
         $this->addDefaultFormTab($ong);
 
         $count = self::getMatchingItemsCount("PluginGlpiinventoryTaskjob");
-        $ong[$this->getType() . '$task'] = self::createTabEntry(_n('Associated task', 'Associated tasks', $count), $count);
+        $ong[$this->getType() . '$task'] = self::createTabEntry(_n('Associated task', 'Associated tasks', Session::getPluralNumber(), 'glpiinventory'), $count);
```

Comment:
You've reverted your change, but you've also updated the other translation to always use `Session::getPluralNumber` - Therefore singular is now no longer used.

Also, not directly related to your PR, but this method does not respect `glpi_count_on_tabs` parameter, I guess it should.

Reply:
This follow the standard of all other tabs on GLPI. Into the "Computer" asset and other pages of GLPI, it always use the plural form to translate the tab names, even if the count shows only one item.

Reply:
To respect parameter, sthing like that should do the trick:
```
$count = 0;
if ($_SESSION['glpishow_count_on_tabs']) {
    $count = self::getMatchingItemsCount("PluginGlpiinventoryTaskjob");
}
```

Reply:
Even with this it's not enough, since the ``glpishow_count_on_tabs`` is true but if there's no item on tab it still shows the string into it's singular form instead of plural.